src/tests/auth/login.test.js

```diff
@@ -0,0 +1,190 @@
+import chaiHttp from 'chai-http';
+import chai, { expect } from 'chai';
+import models from '../../models';
+import app from '../../app';
+import {
+  activeUser,
+  notActive,
+  wrongToken,
+  tokenWithWrongUser,
+  wrongUser
+} from '../fixtures/users.fixture';
+
+chai.use(chaiHttp);
+const router = () => chai.request(app);
+
+const { Users } = models;
+
+let user = null;
+let token = null;
+const userPassword = 'Pass1234';
+
+
+describe('Tests for user login', () => {
+  before(async () => {
+    await Users.create(activeUser);
+    await Users.create(notActive);
+    user = {
+      email: activeUser.email,
+      password: userPassword,
+    };
+  });
+  it('should login a user', (done) => {
+    router()
+      .post('/api/auth/login')
+      .send(user)
+      .end((err, res) => {
+        token = res.body.data;
```

Comment:
A test case should be independent of others. Do not set a global variable in a test case.

Reply:
Then how can I get token of a user who is signed in? because I need to use it to test for verifying a token and accessing protected routes.

Reply:
You manually log the user in wherever you need to use the token

Reply:
let me try it